salt/metalk8s/kubernetes/etcd/health.sls

```diff
@@ -0,0 +1,15 @@
+include:
+  - .installed
```

Comment:
I think it's very strange for a `health` state to `include` `installed`.

For sure this may be renamed to `healthy`. Even then, I'm not sure it should `include` `installed`. We may want to be able to run this state as kind-of an 'assertion' without actually installing/upgrading the node. In the playbook, then apply both SLSs.

Reply:
For me it's more
- we want an etcd pod ok call `etcd.installed`
- we want a healthy etcd pod call `etcd.health(y)` 

For me having a state that only do a check it's strange should be a module at least. 
For me should have this step `Waiting for etcd running` directly in the state that install etcd but etcd may not work directly